stable/redis-ha/templates/redis-ha-configmap.yaml

```diff
@@ -143,5 +143,4 @@ data:
         find_master
     fi
     echo "{{ .Values.redis.masterGroupName }}" > /data/conf/group_name
-    chown -R {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.fsGroup }} /data/
```

Comment:
@khrisrichardson Here it is! I tested this out upgrading from the previous and a fresh install. Both seemed to upgrade smoothly.